Subtester/ProjectSettings/ProjectSettings.asset

```diff
@@ -156,7 +156,7 @@ PlayerSettings:
   androidMaxAspectRatio: 2.1
   applicationIdentifier:
     Android: com.revenuecat.purchases_sample
-    iPhone: com.revenuecat.sampleapp
+    iPhone: com.revenuecat.purchases-sample
```

Comment:
is this an expected change?

Reply:
I think that'd break the iOS app unless we also have one set up in ASC with that bundle id

Reply:
or rather, it'd break the CI that assumes we're using the API key for `sampleapp`

Reply:
I didn't mean to commit that, will revert it. Good catch!